uber/automated_emails.py

```diff
@@ -12,34 +12,69 @@ class AutomatedEmail:
         Attendee: lambda session: session.staffers(only_staffing=False),
         Group: lambda session: session.query(Group).options(subqueryload(Group.attendees))
     }
-    extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, date_filters=[], sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
         self.model, self.template, self.needs_approval = model, template, needs_approval
         self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)
         self.cc = cc or []
         self.bcc = bcc or []
         self.extra_data = extra_data or {}
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
+        self.date_filters = date_filters
```

Comment:
This should probably be

```
self.date_filters = listify(date_filters)
```

That way we could accept either a single date filter or a list of them and our code would work either way.